components/_patterns/02-molecules/address-bar/partials/_address-bar--amenities.scss

```diff
@@ -0,0 +1,21 @@
+.address-bar {
+  &__amenities {
+    @include address-bar-vertical-spacing;
+
+    margin-top: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  &__amenity {
+    position: relative;
+    padding-left: $space-s;
+    letter-spacing: 0.44px;
+
+    &::before {
```

Comment:
<img width="585" alt="Screen Shot 2019-09-09 at 4 45 38 PM" src="https://user-images.githubusercontent.com/1720093/64565216-45cdaf80-d321-11e9-8c78-dbccdf10686c.png">

@jackkim9, is there a reason we're not just using the traditional '·' that you see with an unstyled unordered list? I kept the impression that this design is a result of Figma making it challenging to style something as an unordered list over this being the intended desired visuals, but I could be wrong.

As for why it matters, Brian's had to code around it enough that I think it begs the question.

Reply:
Let's use the traditional '·'